index.js

```diff
@@ -531,6 +531,14 @@ function Argv (processArgs, cwd) {
     return argv
   }
 
+  function guessLocale () {
+    try {
+      return osLocale.sync()
```

Comment:
This is still a bit worrisome for Atom's case where app startup time is critical and this call exec's processes synchronously.